docs/src/concepts.md

```diff
@@ -0,0 +1,36 @@
+
+## Element Architecture
+
+Each of the DataJoint Elements are a set of tables for common neuroinformatics modalities to organize, preprocess, and analyze data. Each node in the following diagram is either a table in the Element itself or a table that would be connected to the Element.
+
+![element-electrode-localization diagram](https://raw.githubusercontent.com/datajoint/element-electrode-localization/main/images/diagram_electrode_localization.svg)
+
+The Element is separated into two schemas:
+
++ `coordinate_framework` ingests standard atlases and retains voxel-based lookup tables, including references to brain region information, acronyms and standardized color codes.
++ `electrode_localization` pairs the above reference tables with Neuropixels probe location data in a `probe` schema, optionally from [Element Array Ephys](https://github.com/datajoint/element-array-ephys)
+
+## Key Partnerships
+
+In coordination with both the Mesoscale Activity Project and the International Brain Lab, DataJoint developed mechanisms for pairing recording coordinates with the published atlases in project-specific cases that have since been generalized.
+
+## Pipeline Development and Limitations
+
+The Element's table architecture was generalized from existing project-specific pipelines to load standardized Kilosort `channel_locations.json` files as well as the Allen Institute's [atlas files](https://community.brain-map.org/t/allen-mouse-ccf-accessing-and-using-related-data-and-tools/359), and pair these data sets.
+
+One notable consession was made in development: acronyms in DataJoint do not perfectly map on to the Allen Institute's published standard. By default, DataJoint databases are not case sensitive. Instead, acronyms are convered to [snake case](https://en.wikipedia.org/wiki/Snake_case) to avoid naming collisions. While we depart from the standard, preliminary interviews with users indicate no bias toward the official standard. Visit our [localization notebook](https://github.com/datajoint/workflow-array-ephys/blob/main/notebooks/08-electrode-localization.ipynb) for a demonstration of converting between the case sensitive and snake case standards. 
+
+## Roadmap
+
+Further development of this Element is community driven. Upon user requests we will continue adding features to this Element, such as improved topological referencing.
```

Comment:
@sidhulyalkar - please confirm the language I suggested here. What I mean is a feature that would allow researchers to better see the relationships between regions and subregions in the ccf hierarchy

Reply:
Yes that sounds appropriate. Could reword it to say "improved regional and subregional based topological referencing" for clarity.